### DISCUSSION TITLE
How to label handles?

### DISCUSSION THREAD

**Discussion by tpoxa** [bcakmakoglu/vue-flow#117] [Q&A] [answered]
I would like to name handles so the user knows to which handle they need to connect an edge.  Any suggestions on how to do that?
Thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
Regular nodes do not support a label on their handles but you could implement a custom node that has a labeled handle.
You can use the exported Handle component to make it easier.

The node will need something like this:
```vue | pure
<Handle
	class="my-custom-handle"
    type="source"
    :position="props.sourcePosition"
    :is-connectable="props.connectable"
    :is-valid-connection="props.isValidSourcePos"
  >
    This is a handle <!-- A label here -->
  </Handle>
```

Add some styles to your Handle and you're done, a labeled handle 😄

**Comment by tpoxa** [comment]
Thanks so much @bcakmakoglu
